### DISCUSSION TITLE
Give the possibility to the SearchResult object as JSON

### DISCUSSION THREAD

**Discussion by carlosmfnt** [spatie/laravel-site-search#15] [Ideas]
I'm implementing this package, and I'm trying to use Vue.js to render the results, for which I need the data to be in JSON format, but when I try to `json_encode()` the SearchResult object, the Hits are returned empty, just like this:

`{"current_page":1,"data":[{},{},{},{},{},{},{},{},{},{}],"first_page_url":"\/?page=1","from":1,"next_page_url":"\/?page=2","path":"\/","per_page":10,"prev_page_url":null,"to":10}`

I think it would be a great addition.

**Comment by freekmurze** [comment]
Feel free to send a PR for this 👍
